app/main.py

```diff
@@ -1 +1,18 @@
-# write your code here
+from app.errors import (NotVaccinatedError,
+                        OutdatedVaccineError,
+                        NotWearingMaskError)
+from app.cafe import Cafe
+
+
+def go_to_cafe(friends: list, cafe: Cafe) -> str:
+    masks_to_buy = 0
+    for friend in friends:
+        try:
+            cafe.visit_cafe(friend)
+        except (NotVaccinatedError, OutdatedVaccineError):
+            return "All friends should be vaccinated"
```

Comment:
The return message 'All friends should be vaccinated' is appropriate for the exceptions caught. Ensure this message aligns with the checklist requirements.